setup.py

```diff
@@ -4,15 +4,8 @@
     from distutils.core import setup
 
 
-install_requires = [
-    "boto3 >= 1.0.1",
-    "pyrax >= 1.9.2",
-    "google-cloud-storage"
-]
-
 setup(name="object_storage",
-      version="0.12.5",
+      version="0.13.0",
       description="Python library for accessing files over various file transfer protocols.",
       url="https://github.com/ustudio/storage",
-      packages=["storage"],
-      install_requires=install_requires)
+      packages=["storage"])
```

Comment:
I think you need to keep the `install_requires` argument because otherwise this library's requirements won't get installed when you do a `pip install object-storage`.